src/App.tsx

```diff
@@ -1,35 +1,55 @@
 import React, { useEffect, useState } from 'react';
-import Feilside from './komponenter/Feilside';
-import NavFrontendSpinner from 'nav-frontend-spinner';
-import Søknad from './komponenter/Søknad';
-import { Panel } from 'nav-frontend-paneler';
+import Banner from './components/Banner';
+import Feilside from './components/feilside/Feilside';
 import hentToggles from './toggles/api';
-import { ToggleName, Toggles } from './typer/toggles';
+import NavFrontendSpinner from 'nav-frontend-spinner';
+import Språkvelger from './components/språkvelger/Språkvelger';
+import Søknad from './søknad/Søknad';
+import { ToggleName, Toggles } from './models/toggles';
+import { hentPersonData } from './utils/søknad';
+import useSøknadContext from './context/SøknadContext';
+import { PersonActionTypes, usePersonContext } from './context/PersonContext';
 
 const App = () => {
   const [toggles, settToggles] = useState<Toggles>({});
   const [fetching, settFetching] = useState<boolean>(true);
-
   const [error, settError] = useState<boolean>(false);
+  const { person, settPerson } = usePersonContext();
+  const { søknad, settSøknad } = useSøknadContext();
 
   useEffect(() => {
     const fetchData = () => {
       hentToggles(settToggles).catch((err: Error) => {
-        settError(true);
+        // settError(true);
       });
+      const fetchPersonData = () => {
+        hentPersonData().then((response) => {
+          settPerson({
+            type: PersonActionTypes.HENT_PERSON,
+            payload: response,
+          });
+          settSøknad({ ...søknad, person: response });
+        });
+      };
+      if (person) {
+        fetchPersonData();
+      }
       settFetching(false);
     };
+
     settError(false);
     fetchData();
+    // eslint-disable-next-line
   }, []);
 
   if (!fetching) {
     if (!error) {
       return (
         <div className="app">
-          <Panel className="innholdspanel">
-            <div>{toggles[ToggleName.vis_innsending] && <Søknad />}</div>
-          </Panel>
+          <Banner tittel={'Enslig forsørger'} />
+          <Språkvelger />
+          <Søknad />
+          {toggles[ToggleName.vis_innsending] && <Søknad />}
```

Comment:
Søknad kommer to ganger. Innenfor og utenfor toggle.

Reply:
Har bare satt den utafor for lokal testing siden jeg ellers ikke får sett søknaden med mindre jeg kjører mottak og api også 😅  😊